HousingSearchApi/V1/Controllers/BaseController.cs

```diff
@@ -35,5 +42,34 @@ public static void ConfigureJsonSerializer()
                 return settings;
             };
         }
+
+        public static string GetErrorMessage(ModelStateDictionary modelState)
```

Comment:
public static string GetErrorMessage(**this** ModelStateDictionary modelState)

Reply:
This make sense, thanks!